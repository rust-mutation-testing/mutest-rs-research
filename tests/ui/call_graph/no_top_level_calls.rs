//@ print-call-graph
//@ print-targets
//@ stdout
//@ stderr: empty

#[test]
fn empty_test() {}

#[test]
fn test_with_basic_stmts() {
    let (_x, _y, _z) = (1, 2, 3);
}
