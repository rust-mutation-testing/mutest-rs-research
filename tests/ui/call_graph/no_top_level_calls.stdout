
@@@ call graph @@@

entry points:

test empty_test
test test_with_basic_stmts

@@@ targets @@@

targets: 0 total; 0 safe; 0 unsafe (0 tainted)
